src/gui/FileBrowser.cpp

```diff
@@ -703,6 +752,25 @@ QList<QAction*> FileBrowserTreeWidget::getContextActions(FileItem* file, bool so
 	return result;
 }
 
+void FileBrowserTreeWidget::addFavorite(QString item)
+{
+	item = ConfigManager::removeTrailingSeparators(item);
+	ConfigManager::addFavoriteItem(item);
+	FileBrowser::getFavoritesBrowserInstance()->reloadTree();
+}
+
+void FileBrowserTreeWidget::removeFavorite(QString item)
+{
+	item = ConfigManager::removeTrailingSeparators(item);
+	ConfigManager::removeFavoriteItem(item);
+	FileBrowser::getFavoritesBrowserInstance()->reloadTree();
+}
+
+bool FileBrowserTreeWidget::isFavorite(QString item)
+{
+	item = ConfigManager::removeTrailingSeparators(item);
+	return ConfigManager::isFavorite(item);
+}
```

Comment:
Function is a one-liner with this suggestion. Actually most of these functions are one liners with the suggestions made, so I would just call `ConfigManager` directly.
```suggestion
bool FileBrowserTreeWidget::isFavorite(const QString& path)
{
	return ConfigManager::isFavorite(item);
}
```